Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1848,11 +1963,29 @@ void MediaPlayerPrivateGStreamer::cacheDuration()
     m_mediaDuration = newDuration;
 }
 
+static gboolean mediaPlayerPrivateNotifyDurationChanged(gpointer userdata)
```

Comment:
You can have the MediaPlayerPrivateGStreamer pointer directly here instead of a gpointer I think.